iron-inttest/src/test/java/io/axway/iron/spi/file/FileStoreTest.java

```diff
@@ -26,15 +27,18 @@
 
 import static io.axway.iron.spi.file.FileTestHelper.*;
 import static io.axway.iron.spi.jackson.JacksonTestHelper.*;
+import static java.nio.file.Paths.get;
+import static java.util.Arrays.*;
 import static java.util.Comparator.*;
+import static java.util.stream.Collectors.*;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class FileStoreTest {
 
     @Test
     public void shouldCreateCompanySequenceBeRight() throws Exception {
         String randomStoreName = "iron-store-" + UUID.randomUUID();
-        Path filePath = Paths.get("tmp-iron-test", "iron-spi-file-inttest");
+        Path filePath = get("tmp-iron-test", "iron-spi-file-inttest");
```

Comment:
in this case, static import make the code less readable

Reply:
You're right.